packages/ra-ui-materialui/src/input/ArrayInput.spec.js

```diff
@@ -1,45 +1,17 @@
 import React from 'react';
 import { shallow, mount } from 'enzyme';
+import { render, waitForDomChange } from 'react-testing-library';
 import { reduxForm } from 'redux-form';
 import { TestContext } from 'ra-core';
 
-import ArrayInput, { ArrayInput as ArrayInputView } from './ArrayInput';
+import ArrayInput from './ArrayInput';
 import NumberInput from './NumberInput';
 import TextInput from './TextInput';
 import SimpleFormIterator from '../form/SimpleFormIterator';
 
 describe('<ArrayInput />', () => {
-    it('should render a FieldArray', () => {
-        const wrapper = shallow(<ArrayInputView source="arr" record={{}} />);
-        expect(wrapper.find('pure(FieldTitle)').length).toBe(1);
-        expect(wrapper.find('FieldArray').length).toBe(1);
-    });
-
-    it('should pass an undefined fields to child when initialized with an undefined value', () => {
```

Comment:
why did you remove this test?

Reply:
Because it tests redux-form, not react-admin